backend/pom.xml

```diff
@@ -101,9 +101,10 @@
 		</dependency>
 
 		<dependency>
-			<groupId>org.postgresql</groupId>
-			<artifactId>postgresql</artifactId>
+			<groupId>org.mariadb.jdbc</groupId>
```

Comment:
Soll das wirklich als mariadb in den branch gemerget werden? Die meisten nutzen doch PostgreSQL oder lieg ich da falsch?
Macht es dann nicht mehr Sinn diese Änderung nicht zu mergen?